certification/internal/shell/has_v2_related_image_schema_version.go

```diff
@@ -19,192 +12,16 @@ type RelatedImagesAreSchemaVersion2Check struct{}
 // Validate checks the image manifest for each related image referenced in a
 // ClusterServiceVersion and ensures that the schema version used is version 2.
 func (p *RelatedImagesAreSchemaVersion2Check) Validate(image string) (bool, error) {
-	imageToSchemaVersion, err := p.getDataToValidate(image)
-	if err != nil {
-		return false, fmt.Errorf("%w: %s", errors.ErrRunContainerFailed, err)
-	}
-
-	return p.validate(imageToSchemaVersion)
-}
-
-// getDataToValidate pulls a ClusterServiceVersion from the input operator bundle,
-// checks the ClusterServiceVersion's related images declaration, and assembles a list of
-// images and their image manifest schema version values.
-func (p *RelatedImagesAreSchemaVersion2Check) getDataToValidate(image string) (map[string]int, error) {
-	bundlePath, err := p.extractBundleFromImage(image)
-	if err != nil {
-		log.Error(err)
-		return nil, fmt.Errorf("%w: %s", errors.ErrDeterminingRelatedImageSchemaVers, err)
-	}
-	csv, err := p.readBundle(bundlePath)
+	mounted := true
```

Comment:
Should we just pass `true` into `UnshareWithCheck` and leave out the variable assignment?

Reply:
I don't like bare booleans. It makes it explicit what the called function is expecting. And it shouldn't affect performance in any way, as I'm quite sure the compiler would just optimize this out.

Reply:
Ah, I can get behind that. 👍